src/main/java/cloud/orbit/web/EmbeddedJettyServer.java

```diff
@@ -229,6 +235,14 @@ public Task start()
             {
                 sslContextFactory.setExcludeCipherSuites(sslExcludedCipherSuites.toArray(new String[0]));
             }
+            if (sslIncludedProtocols != null)
```

Comment:
Good catch! I made a change to initialize the config fields to null so as not to overwrite sslContextFactory's defaults.